requirements.txt

```diff
@@ -16,3 +16,6 @@ black
 pytest
 coverage
 pytest-cov
+folium
+geopandas
+rtree
```

Comment:
We'll want to add these to the Pipfile so that we can sync all necessary dependencies.